config/locales/views/en.yml

```diff
@@ -1358,7 +1358,18 @@ en:
         what_you_cant_do: What you can't do
   invitations:
     invitation:
+      copy_and_use: copy and use
+      copy_link: Copy link
+      created_at: Created at
+      deleted_user: deleted user (%{user_id})
```

Comment:
Can you capitalize "Deleted" here so the invitee and sender will match when we do [AO3-6095](https://otwarchive.atlassian.net/browse/AO3-6095)? I was going to say I'd add the user ID to that issue, but I think it would show the ID to users, not just admins, so we probably don't want that. 

Relatedly, are we sure this only shows the user ID if you're an admin?

Reply:
Oh at the moment it definitely shows to both, my thought was it's not a big concern because user ID isn't PII. But it's also not particularly _useful_ to a non-admin so I am also not opposed to adding another check in the helper

Reply:
Let's go with an additional check, then! I'll update AO3-6095 to request the same behavior.